code/modules/cooking/recipes/recipes_fryer.dm

```diff
@@ -167,3 +167,18 @@
 	)
 	coating = /singleton/reagent/nutriment/coating/batter
 	result = /obj/item/reagent_containers/food/snacks/corn_dog
+
+/singleton/recipe/north60squid
+	appliance = FRYER
+	items = list(/obj/item/reagent_containers/food/snacks/squidmeat = 1)
+	reagents = list(/singleton/reagent/drink/lemonjuice = 5 , /singleton/reagent/drink/applejuice = 5, /singleton/reagent/nutriment/garlicsauce = 10)
+	coating = /singleton/reagent/nutriment/coating/beerbatter
+	reagent_mix = RECIPE_REAGENT_REPLACE //Simplify end product
+	result = /obj/item/reagent_containers/food/snacks/north60squid
+	//XANAN
```

Comment:
?

Reply:
What's confusing? The annotation at the end? That's my bad, I thought it was a way of designating the origin of a dish before I realized how it's actually categorized.

Reply:
Fixed. Thank you!